odoo/addons/base/data/res_currency_data.xml

```diff
@@ -1687,34 +1607,146 @@
             <field name="currency_subunit_label">Centavo</field>
         </record>
 
-        <record id="SKK" model="res.currency">
-            <field name="name">SKK</field>
-            <field name="full_name">Slovak koruna</field>
-            <field name="symbol">Sk</field>
+        <record id="GTQ" model="res.currency">
+            <field name="name">GTQ</field>
+            <field name="full_name">Guatemalan quetzal</field>
+            <field name="symbol">Q</field>
+            <field name="rounding">0.01</field>
+            <field name="active" eval="False"/>
+            <field name="currency_unit_label">Quetzal</field>
+            <field name="currency_subunit_label">Centavo</field>
+        </record>
+
+        <record id="XDR" model="res.currency">
+            <field name="name">XDR</field>
+            <field name="full_name">Special drawing rights</field>
+            <field name="symbol">SDR</field>
+            <field name="active" eval="False"/>
+        </record>
+
+        <record id="COU" model="res.currency">
+            <field name="name">COU</field>
+            <field name="full_name">Unidad de Valor Real</field>
+            <field name="symbol">$</field>
             <field name="rounding">0.01</field>
             <field name="active" eval="False"/>
-            <field name="currency_unit_label">Koruna</field>
-            <field name="currency_subunit_label">Halier</field>
+            <field name="position">before</field>
+            <field name="currency_unit_label">Peso</field>
+            <field name="currency_subunit_label">centavo</field>
         </record>
 
-        <record id="YUM" model="res.currency">
-            <field name="name">YUM</field>
-            <field name="full_name">Yugoslav dinar</field>
-            <field name="symbol">дин.</field>
+        <record id="CLF" model="res.currency">
+            <field name="name">CLF</field>
+            <field name="full_name">Unidad de Fomento</field>
+            <field name="symbol">$</field>
+            <field name="rounding">0.0001</field>
+            <field name="active" eval="False"/>
+            <field name="position">before</field>
+            <field name="currency_unit_label">Peso</field>
+            <field name="currency_subunit_label">Centavos</field>
+        </record>
+
+        <record id="BOV" model="res.currency">
+            <field name="name">BOV</field>
+            <field name="full_name">Bolivian Mvdol (funds code)</field>
```

Comment:
```suggestion
            <field name="full_name">Bolivian Mvdol</field>
```

Reply:
In fact, such special currencies shouldn't be added. See my comment on USN below.